src/pages/Regions/AddRegionSecretaryForm.tsx

```diff
@@ -25,18 +25,11 @@ const AddNewSecretaryForm = (props: any) => {
   const [startDate, setStartDate] = useState<any>();
   const [users, setUsers] = useState<any[]>([
```

Comment:
Maybe it would be better to create a model for the user or use existing instead of hardcoding it there?
```suggestion
  const [users, setUsers] = useState<User[]>();
```